### PR TITLE
Update validators.md

### PR DESCRIPTION
**Pull request by ishanray**
I was confused over how to actually run the validation so adding it here in the docs.

## Description

Fixes (issue #)

## Type of change

- [ ]  Patch: Bug (non-breaking change which fixes an issue)
- [ ]  Minor: New feature (non-breaking change which adds functionality)
- [ ]  Major: Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] I have added a changelog entry, prefixed by the type of change noted above (Documentation fix and Test update does not need a changelog as we do not publish new version)

### CODE REVIEW

File: packages/react-form-state/docs/validators.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ import {validators} from '@shopify/react-form-state';
 
 We can pass these directly into the `validators` prop of `<FormState />`.
 
+To validate the form automatically on form submit set the `validateOnSubmit` prop of `FormState` to be `true`.
```
Comment (gmcgibbon) [inline packages/react-form-state/docs/validators.md:26]:
```suggestion
To use validators when submitting your form, set the `validateOnSubmit` prop of `<FormState />` to `true`. Here's an example:
```

I would also remove the line break to make a paragraph out of the sentence above, and then edit the example below to use this property.

File: packages/react-form-state/docs/validators.md [hunk 2]
```diff
@@ -35,6 +36,7 @@ function MyComponent() {
         description: 'Cool product',
         quantity: 0,
       }}
+      validateOnSubmit
```
Comment (gmcgibbon) [inline packages/react-form-state/docs/validators.md:39]:
```suggestion
      validateOnSubmit={true}
```
or is just specifying the property magic syntax for the same thing?

Comment (ishanray) [reply packages/react-form-state/docs/validators.md:39]: Yup it is for the same thing, just some syntactical sugar.

**Review by gmcgibbon — APPROVED** [review]
:shipit:

**Pull request merged**